src/assets/i18n/en.json

```diff
@@ -981,5 +981,32 @@
     "text-2": "Nice and sound",
     "text-3": "Good job!",
     "text-finish": "Thank you !!!"
+  },
+  "create-event": {
+    "back": "назад",
+    "title": "Create Event",
+    "edit": "Edit Event",
+    "description": "Please provide as much details as you can - place and time of event, the goal of gathering etc. You can come back and update event anytime after publishing",
+    "event-name": "Enter name for Event",
+    "event-name-length": "/70",
```

Comment:
Need to remove "event-name-length": "/70", "picture-length": "/5" and "picture-delete": "X" because we don't need translation for it